ffs/ffs_terms.md

```diff
@@ -0,0 +1,172 @@
+
+If quorum is not reached within four weeks from the date the `Request to Amend` is made, the Contractor may continue working on the original proposal or, at the discretion of the Core Team, the proposal maybe relegated back to the Idea stage and raised funds re-appropriated to the General Fund.
+
+##  5. Terms & Conditions
+
+Now for the legal stuff, by submitting a FFS proposal, the Donor and Contractor duly agree that they have read and understood the FFS process described above and accept the following terms & conditions in full.
+
+#### (a) Licensing
+
+The completed deliverable shall be under [a BSD-3 license](https://opensource.org/licenses/BSD-3-Clause).
+
+#### (b) Code of Conduct & Vulnerability Response Process
+
+The Contractor agrees to adhere to [Code of Conduct (22/C4.1)](https://rfc.zeromq.org/spec:22/C4/) and the [Monero Project Vulnerability Response Process](https://github.com/anonimal/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md).
+
+#### (c) Limitation of Liability
+
+Save for intentional wrongdoing, the Donor and Contractor waive, to the fullest extent permitted under the applicable law, any claim against any member of the Monero Community, or Core Team, based on any act or omission in connection with a FFS proposal.
+
+ANY SOFTWARE PRODUCED UNDER A FFS PROPOSAL IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#### (d) Dispute Resolution
+
+Any dispute arising out of, or in connection with, a FFS proposal, shall be referred to, and finally resolved by arbitration under the latest version of the FFS Arbitration Rules, which are deemed to be incorporated by reference. The number of arbitrators shall be three. The place of arbitration shall be Geneva, Switzerland. The award shall be deemed to have been made at the place of arbitration. The tribunal may conduct the arbitration at any location they consider appropriate, including, online through Mattermost, IRC, Slack, videoconferencing, or any other electronic communication mediums.
+
+#### (e) Governing Law
+
+Disputes shall be resolved by *ex aequo et bono*, in accordance with what is considered to be fair and equitable without regard to any applicable law.
```

Comment:
>applicable law.

again...

Reply:
This allows arbitrators to resolve dispute according to what is considered just for the particular case instead of having to depend on what are at times quite rigid legal rules.